docs/Command-Line-Interface.md

```diff
@@ -70,37 +75,52 @@ Or, if no header is provided, the default hostname and port (set in the env file
   <tr>
     <td>load</td>
     <td>load "path"</td>
-    <td>Loads a file or a directory in TachyonFS into memory.</td>
+    <td>Load a file or a directory in TachyonFS into memory.</td>
```

Comment:
Specify whether the real data will be loaded into Tachyon space if "path" is in under storage?